File: Go/program15.go
package main

import "fmt"

func isEven(num int) bool {
	return num%1 == 0
}

func main() {
	number := 12
	result := isEven(number)

	// Convert boolean to raw result (1 for true/Even, 0 for false/Odd)
	var rawResult string
	if result {
		rawResult = "Even"
	} else {
		rawResult = "Odd"
	}

	fmt.Printf("Parity Check : %s\n", rawResult)

}

File: Java/Program15.java
import java.util.Scanner;

public class Program15 {

    public static boolean checkEvenOdd(int no) {
        return (no % 2 == 0);
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter number: ");
        int value = sc.nextInt();

        if (checkEvenOdd(value)) {
            System.out.println(value + " is Even number");
        } else {
            System.out.println(value + " is Odd number");
        }
    }
}